### DISCUSSION TITLE
My GitHub page, for a project's documentation, won't load

### DISCUSSION THREAD

**Discussion by ghost** [community/community#22904] [Other Feature Feedback, Questions, & Ideas]
<p>I have a <a href="https://github.com/Kove-W-O-Salter/Cyp" rel="nofollow noopener">project</a>, which I would like to create some documentation for. I have created a directory in the project’s root, on the master branch, called:</p>
<pre><code>docs/ 
</code></pre>
<p>It contains around 190 KBs of documentation, generated by Haddock. I have enabled GitHub Pages for this repository. Here is my projects root directory:</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/1X/a97365620f9cf0f4aa66ce7ea31c2559fa7ca54c.png" data-download-href="/uploads/short-url/ob1VuKA6hhUrcME63RXylweopze.png?dl=1" title="image.png" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/10137/181073677-a48a5331-d2e0-49dc-8e88-c6012dc45962.png" alt="image.png" data-base62-sha1="ob1VuKA6hhUrcME63RXylweopze" width="690" height="387" srcset="https://aws1.discourse-cdn.com/github/optimized/1X/a97365620f9cf0f4aa66ce7ea31c2559fa7ca54c_2_690x387.png, https://aws1.discourse-cdn.com/github/optimized/1X/a97365620f9cf0f4aa66ce7ea31c2559fa7ca54c_2_1035x580.png 1.5x, https://aws1.discourse-cdn.com/github/original/1X/a97365620f9cf0f4aa66ce7ea31c2559fa7ca54c.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/1X/a97365620f9cf0f4aa66ce7ea31c2559fa7ca54c_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">image.png</span><span class="informations">1366×768 82.6 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>Here is an image of my current configuration:</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/1X/7dd4857ec87f846e7b16b020a8fe7106d20c22bd.png" data-download-href="/uploads/short-url/hX90yofgUqg6f6rmkYqiaecDsx7.png?dl=1" title="image.png" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/10137/181073678-c792f394-7ad0-45bb-9db5-ed2fde5d2cf1.png" alt="image.png" data-base62-sha1="hX90yofgUqg6f6rmkYqiaecDsx7" width="690" height="387" srcset="https://aws1.discourse-cdn.com/github/optimized/1X/7dd4857ec87f846e7b16b020a8fe7106d20c22bd_2_690x387.png, https://aws1.discourse-cdn.com/github/optimized/1X/7dd4857ec87f846e7b16b020a8fe7106d20c22bd_2_1035x580.png 1.5x, https://aws1.discourse-cdn.com/github/original/1X/7dd4857ec87f846e7b16b020a8fe7106d20c22bd.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/1X/7dd4857ec87f846e7b16b020a8fe7106d20c22bd_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">image.png</span><span class="informations">1366×768 67.4 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>When I follow the URL that my page should be on (the one specified in the settings page), I just get a 404 error saying that the page cannot be found:
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/1X/ce83af30f1e70d1abfd89b8ddc7724e885392127.png" data-download-href="/uploads/short-url/tsUusBaZGi0XWLwvLMQ2PgRiptZ.png?dl=1" title="image.png" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/10137/181073679-1d2d9d85-e44a-4c8f-a998-493bc5be03ef.png" alt="image.png" data-base62-sha1="tsUusBaZGi0XWLwvLMQ2PgRiptZ" width="690" height="387" srcset="https://aws1.discourse-cdn.com/github/optimized/1X/ce83af30f1e70d1abfd89b8ddc7724e885392127_2_690x387.png, https://aws1.discourse-cdn.com/github/optimized/1X/ce83af30f1e70d1abfd89b8ddc7724e885392127_2_1035x580.png 1.5x, https://aws1.discourse-cdn.com/github/original/1X/ce83af30f1e70d1abfd89b8ddc7724e885392127.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/1X/ce83af30f1e70d1abfd89b8ddc7724e885392127_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">image.png</span><span class="informations">1366×768 23.8 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>This is still happening around 20 minutes after I enabled GitHub Pages for that repository. Can someone please tell me what I’m doing wrong?

**Comment by ghost** [comment]
<p>I can access it by appending</p>
<pre><code>index.html
</code></pre>
<p> to the URL. Does anyone know how to fix this?</p>

**Comment by Dextrices** [comment]
<p>From the looks of it, you are using the branch “master” instead of “gh-pages”. I think you have to use the gh-pages branch in order for it to act like a webpage. </p>

**Comment by oycyc** [comment, marked as answer]
<p>From my browser, <a href="https://kove-w-o-salter.github.io/Cyp" rel="nofollow noopener">https://kove-w-o-salter.github.io/Cyp</a> works fine. I’m not sure why your browser doesn’t work, but I think it didn’t update yet, so you might have to clear cookies and cache or try a new browser.</p>

**Comment by ghost** [comment]
<p>Thanks, heaps <a class="mention" href="/yangciou">@yangciou</a>. I restarted my browser and now it is working fine.</p>
